src/main/config/secor.common.properties

```diff
@@ -181,10 +181,17 @@ kafka.dual.commit.enabled=true
 # Possible values: "zookeeper" to read offset from zookeeper or "kafka" to read offset from kafka consumer topic
 kafka.offsets.storage=zookeeper
 
-# Parameter which tells whether to extract Kafka message timestamp. This value is to be chose in case of 0.10.x kafka
-# brokers. Default value is false.
+# Parameter which tells whether to extract Kafka message timestamp. This value is to be chose in case of 0.10.x kafka brokers.
+# Default value is false. Also specify `kafka.message.timestamp.className` as `com.pinterest.secor.timestamp.Kafka10MessageTimestamp`,
+# in case you are enabling this parameter as `true`.
 kafka.useTimestamp=false
 
+# Classname for the timestamp field you want to use.
+# Basically, it will be `Kafka8MessageTimestamp` for 0.8 kafka and `Kafka10MessageTimestamp` for 0.10 kafka,
+# Fully classified names are `com.pinterest.secor.timestamp.Kafka8MessageTimestamp` and
+# `com.pinterest.secor.timestamp.Kafka10MessageTimestamp`.
+kafka.message.timestamp.className=com.pinterest.secor.timestamp.Kafka8MessageTimestamp
```

Comment:
I think the default value should be Kafka10MessageTimestamp since the default build profile is now kafka 10.